assets/helpers/tracking/acquisitions.js

```diff
@@ -32,6 +32,7 @@ export type ReferrerAcquisitionData = {|
   componentType: ?string,
   source: ?string,
   abTest: ?AcquisitionABTest,
+  abTests: ?Array<AcquisitionABTest>,
```

Comment:
What are the differences between `abTest` and `abTests` fields?

Reply:
The idea of this type is to mirror the type defined [here](https://goo.gl/M9PtYa). Currently, that type doesn't have a `abTests`field. Is it necessary to add this piece of information here?